### DISCUSSION TITLE
Would you accept a pull request that allows configuring process-compose command-line arguments?

### DISCUSSION THREAD

**Discussion by VanCoding** [Platonic-Systems/process-compose-flake#76] [Ideas]
My goal is to be able to use the new  [--detached](https://f1bonacc1.github.io/process-compose/cli/process-compose/) option. But currently, there's no way to configure command-line arguments.

I initially thought about having an "extraCommandLineArguments" string option, that's then taken into account [here](https://github.com/Platonic-Systems/process-compose-flake/blob/dcf3082c74e944e5d260a6d61780aaf3aff179c8/nix/process-compose/cli.nix#L51), but that would not work well when different modules want to set different arguments.

So maybe I'd add an option for every argument there currently is, and an "extraArguments" options for stuff that gets added later and is not implemented. Then, these options could also be used by the httpServer options.

Would you accept such a pull request?

**Comment by srid** [comment]
Yes, @VanCoding - such a PR would be welcome.

There already is a CLI module that you can contribute to.

https://github.com/Platonic-Systems/process-compose-flake/blob/main/nix/process-compose/cli.nix
